server/neptune/workflows/internal/notifier/github.go

```diff
@@ -128,21 +128,29 @@ type checkRunClient interface {
 
 type CheckRunNotifier struct {
 	CheckRunSessionCache checkRunClient
+	Mode                 string
+}
+
+type Info struct {
+	ID       uuid.UUID
+	Commit   github.Commit
+	RootName string
+	Repo     github.Repo
 }
 
-func (n *CheckRunNotifier) Notify(ctx workflow.Context, deploymentInfo terraform.DeploymentInfo, workflowState *state.Workflow) error {
-	return errors.Wrap(n.updateCheckRun(ctx, workflowState, deploymentInfo), "updating check run")
+func (n *CheckRunNotifier) Notify(ctx workflow.Context, info Info, workflowState *state.Workflow) error {
+	return errors.Wrap(n.updateCheckRun(ctx, workflowState, info), "updating check run")
 }
 
-func (n *CheckRunNotifier) updateCheckRun(ctx workflow.Context, workflowState *state.Workflow, deploymentInfo terraform.DeploymentInfo) error {
+func (n *CheckRunNotifier) updateCheckRun(ctx workflow.Context, workflowState *state.Workflow, info Info) error {
 	summary := markdown.RenderWorkflowStateTmpl(workflowState)
 	checkRunState := determineCheckRunState(workflowState)
 
 	request := GithubCheckRunRequest{
-		Title:   terraform.BuildCheckRunTitle(deploymentInfo.Root.Name),
-		Sha:     deploymentInfo.Commit.Revision,
+		Title:   BuildCheckRunTitle(n.Mode, info.RootName),
```

Comment:
i don't really like `atlantis/pr: <root>` as the check run title.  instead can it be like `atlantis/plan...` or `atlantis/build..` When you click it in you can see the plan and the validation steps.